src/components/Card/Card.tsx

```diff
@@ -27,23 +25,18 @@ const Card: React.FC<ICardProps> = ({
   releaseDate,
   genre,
   id,
-  setActiveMovie,
 }) => {
   const releaseYear = useMemo(() => new Date(releaseDate).getFullYear(), [
     releaseDate,
   ]);
 
-  const selectFilm = useCallback(() => {
-    setActiveMovie(id);
-  }, [id]);
-
   return (
     <Composition as={StyledCard} templateRows={"28.75rem 1fr"}>
       <StyledImageWrapper>
         <CardOptions id={id} />
-        <a href="#" onClick={selectFilm}>
+        <Link to={`/film/${id}`}>
```

Comment:
it's a good practice to move routes to constant and use them between the components.